utils/checks.stanza

```diff
@@ -236,22 +252,26 @@ defn cmos-and-ttl-checks (out-p:JITXObject,
 ;
 defn open-collector-checks (io-pin:JITXObject, 
```

Comment:
Do not call this 100 times for a connected group with 100 io pins + we are missing a pin in the input (original p)